index.html

```diff
@@ -4,9 +4,48 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:wght@100;300;400&family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400&family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Setup</title>
 </head>
 <body>
-    
+    <nav class="app-bar">
+        <div class="logo-placeholder">
+            <h1 class="logo">My Logo</h1>
+        </div>
+        <div class="menu-container">
+            <img class="menu" src="Menu\Tertiary\Icons\Icon - Menu.png" alt="Menu icon">
+        </div>
+    </nav>
+    <section class="header">
+        <div class="text-container">
+            <h1 class="hello">Hi There. I'm Johnson.</h1>
+        </div>
+    <div class="subject">
+        <h2 class="john">I'm a Software Developer</h2>
+    </div>
+    <div class="description container">
+        <p class="dText">
+            I can help you build a product,
+            feature or website. Look through
+            some of my work and experience!
+            If you like what you see and have
+            a project you need coded,
+            don’t hesitate to contact me.
+        </p>
+    </div>
+    <div class="SM-Container">
+        <div class="box"><img src="Icon\GitHub\Tertiary\Icons\Icon -GitHub.png" alt="github icon"></div>
+        <div class="box"><img src="Icon\LinkedIn\Tertiary\Icons\Icon - Linkedin.png." alt="linkedin icon"></div>
+        <div class="box"><img src="Icon\Twitter\Tertiary\Icons\Icon - Twitter.png" alt="twitter icon"></div>
+        <div class="box"><img src="Icon\Medium\Tertiary\Icons\Icon - Medium.png" alt="medium icon"></div>
+        <div class="box"><img src="Icon\Angelist\Tertiary\Icons\Icon - Angelist.png" alt="angelist icon"></div>
+    </div>
```

Comment:
- [ ] I think your social media links should be a list of icons a use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).  Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href Like this
  ```html
   <ul>
        <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
        <!-- other links here -->
   </ul>
  ```